src/zope/publisher/tests/test_http.py

```diff
@@ -487,8 +487,10 @@ def testCookieErrorToLog(self):
             handler.uninstall()
 
         self.assertEqual(len(handler.records), 1)
-        self.assertEqual(handler.records[0].getMessage(),
-                         'Illegal key value: ldap/OU')
+
+        message = handler.records[0].getMessage()
+        self.assertTrue(message.startswith('Illegal key'))
+        self.assertTrue('ldap/OU' in message)
```

Comment:
I would've liked to see what the message looks like on Python 3.5 somewhere (in the commit message, or in a comment), but eh.

LGTM.